design/sketch.webidl

```diff
@@ -755,11 +754,11 @@ interface GPUSwapChain {
 
 // Web GPU "namespace" used for device creation
 dictionary GPUExtensions {
-    boolean anisotropicFiltering;
+    boolean anisotropicFiltering = false;
 };
 
 dictionary GPULimits {
-    u32 maxBindGroups;
+    u32 maxBindGroups = 4;
```

Comment:
This value is filled in by the implementation, not the user, right? It having a default doesn't make sense to me -- it just means a user can do "new GPULimits()" and get confused when maxBindGroups is 4, which might not match the implementation.

Reply:
Limits will be passed in device creation in GPUDeviceDescriptor too. For at least some of these fields (like maxBindGroups), an unspecified limit should mean default limit, not system max limit, for better portability. Apps can easily opt into higher capabilities.

Reply:
BTW, I'm pretty sure you can't construct dictionaries (`new GPULimits()`).